k9mail-library/src/main/java/com/fsck/k9/mail/store/imap/ImapStore.java

```diff
@@ -2878,7 +2879,7 @@ public Object foundLiteral(ImapResponse response,
                 String contentType = mPart
                         .getHeader(MimeHeader.HEADER_CONTENT_TYPE)[0];
 
-                return MimeUtility.decodeBody(literal, contentTransferEncoding,
+                return MimeUtility.createBody(literal, contentTransferEncoding,
```

Comment:
I guess a merge makes sense. If the change didn't predate your refactoring the code would have probably ended up in `MimeMessageHelper`.